src/com/tony/server/WorkerThread.java

```diff
@@ -8,19 +8,28 @@
 public class WorkerThread implements Runnable {
 
     private final ResponseDeterminer responseDeterminer;
+    private final Logger logger;
     private Socket clientSocket;
 
-    public WorkerThread(Socket clientSocket, ResponseDeterminer responseDeterminer) throws IOException {
+    public WorkerThread(Socket clientSocket, ResponseDeterminer responseDeterminer, Logger logger) throws IOException {
         this.clientSocket = clientSocket;
         this.responseDeterminer = responseDeterminer;
+        this.logger = logger;
     }
 
     @Override
     public void run() {
         try {
             Request request = RequestParser.parseRequest(getInputStream());
+            logger.storeLog("Received Request:\n");
```

Comment:
I'm a little confused by this. I was just curious if you intended to have two new lines after 24 and 29, since `#storeLog` already appends a new line.